kolena/asset.py

```diff
@@ -145,4 +146,20 @@ def __post_init__(self) -> None:
             raise ValueError(f"Specified start time '{self.start}' and end time '{self.end}' must be non-negative")
 
 
-_ASSET_TYPES = [ImageAsset, PlainTextAsset, BinaryAsset, PointCloudAsset, BaseVideoAsset, VideoAsset]
+@dataclass(frozen=True, config=ValidatorConfig)
+class AudioAsset(Asset):
+    """
+    An audio file in a cloud bucket or served at a URL.
+
+    For best results, use a broadly supported file type such as `.mp3` or `.wav`.
+    """
+
+    locator: str
+    """The location of this text file in a cloud bucket, e.g. `s3://my-bucket/path/to/my-audio-asset.mp3`."""
```

Comment:
```suggestion
    """The location of this audio file in a cloud bucket, e.g. `s3://my-bucket/path/to/my-audio-asset.mp3`."""
```

Reply:
Thanks for catching, @yifanwu-kolena. Let's see if GitHub has fixed their commit-from-UI behavior around newlines at EOF.